src/controllers/vocabulary.controller.ts

```diff
@@ -0,0 +1,82 @@
+import { Request, Response } from 'express';
+import { vocabularyService } from '../services/vocabulary.service';
+import { IVocabulary } from '../interfaces/vocabulary.interface';
+import HttpException from '../application/utils/exceptions/http-exceptions';
+import HttpStatusCode from '../application/utils/exceptions/statusCode';
+
+class VocabularyController {
+  async getById(req: Request, res: Response): Promise<void> {
+    const vocabularyId: string = req.params.id;
+
+    try {
+      const vocabulary = await vocabularyService.getById(vocabularyId);
+      res.status(200).json(vocabulary);
+    } catch (error) {
+      if (error instanceof HttpException) {
+        res.status(error.status).json({ error: error.message });
+      } else {
+        res.status(HttpStatusCode.INTERNAL_SERVER_ERROR).json({
+          error: 'Internal Server Error',
+        });
+      }
+    }
+  }
+
+  async create(req: Request, res: Response): Promise<void> {
+    const newVocabulary = req.body;
```

Comment:
I am in state of a bit unsure, as it may not be the issue of "todays agenda", however  directly using req.body without sanitization can lead to security issues.

Reply:
For now I think we can not spend so much time on validation and take a look on business logic, and after everything is complete, we can then proceed with tests and validation of requests